sysl2/sysl/tests/integration_with_restrict_by.sysl

```diff
@@ -0,0 +1,19 @@
+IntegratedSystem [appfmt="%(appname)", epfmt="%(epname)", r="y"]:
```

Comment:
Use the following file
```
IntegratedSystem:
    integrated_endpoint_1:
        Systema <- endpoint
    integrated_endpoint_2:
        Systemb <- endpoint

Systema [foo_id="y"]:
    endpoint [foo_id="y"]:
        Systemb <- endpoint2

    endpoint_2 [foo_id="y"]:
        Systemb <- endpoint2

Project [appfmt="%(appname)", epfmt="%(epname)"]:
    with_restrict_by [restrict_by="foo_id"]:
        IntegratedSystem

    _  :
        IntegratedSystem

Systemb:
    endpoint: ...
    endpoint2: ...
```

see the output between the two images using python version.